eng/pipelines/libraries/run-test-job.yml

```diff
@@ -173,17 +173,16 @@ jobs:
               - gcstress0xc_jitstress2
               - gcstress0xc_jitminopts_heapverify1
             ${{ if in(parameters.coreclrTestGroup, 'pgo') }}:
-              scenarios:
-              - nopgo
-              - defaultpgo
-              - dynamicpgo
-              - fullpgo
-              - fullpgo_methodprofiling
-              - fullpgo_random_gdv
-              - fullpgo_random_gdv_methodprofiling_only
-              - fullpgo_random_edge
-              - fullpgo_random_gdv_edge
-              - jitosr_stress
-              - jitosr_stress_random
-              - jitosr_pgo
-
+              ${{ if and(eq(parameters.osGroup, 'windows'), or(eq(parameters.archType, 'arm64'), eq(parameters.archType, 'arm'))) }}:
+                scenarios:
+                - defaultpgo
+              ${{ else }}:
+                scenarios:
+                - fullpgo
+                - fullpgo_methodprofiling
+                - fullpgo_random_gdv
+                - fullpgo_random_gdv_methodprofiling_only
+                - fullpgo_random_edge
```

Comment:
I need to add `defaultpgo` here.

@AndyAyersMS, what do you think about removing `fullpgo_random_edge` in favor of just `fullpgo_random_gdv_edge`? After adding `defaultpgo` that will keep us at 8 scenarios which should shorten the time for this pipeline to run since 4 scenarios can be queued in parallel.

Reply:
That would be ok.